app/views/accounts/_edit_form.html.erb

```diff
@@ -1,49 +0,0 @@
-<%= form_for(@account, class: 'form') do |f| %>
```

Comment:
Yes, there is an `edit` form.  The partial is unused.  It was already deleted once but seems to have crept back in via rebase.  See #655.